cmd/gin-repod/repo.go

```diff
@@ -749,27 +749,29 @@ func (s *Server) listRepoCollaborators(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	type collaborator struct {
+		User        string `json:"User"`
+		AccessLevel string `json:"AccessLevel"`
```

Comment:
One could implement the `Marshaler` interface for `store.AccessLevel` type and then could use it here directly instead of the `string` type.